en/administration/secure-platform.md

```diff
@@ -843,7 +843,7 @@ and enable for **IPv4** inputs and outputs:
 This the official [Centreon gorgone documentation](https://github.com/centreon/centreon-gorgone/blob/master/docs/configuration.md#gorgonecore)
 to secure the communication.
 
-## Security event information management - SEIM
+## Security Information and Event Management - SIEM
```

Comment:
could update french acronym too ? https://github.com/centreon/centreon-documentation/blob/cc9b62469fee35c72206ea9b44722f8190a144b1/fr/administration/secure-platform.md#gestion-des-informations-sur-les-%C3%A9v%C3%A9nements-de-s%C3%A9curit%C3%A9---seim

Reply:
Hello, French version has been updated according the English version.